src/lib.rs

```diff
@@ -4,5 +4,10 @@
 
 pub mod config;
 pub mod errors;
+//Export crate for re-use in project
+pub extern crate aws_config;
+pub extern crate aws_types;
+
 #[cfg(feature = "dynamodb")]
-pub mod dynamodb;
\ No newline at end of file
+pub mod dynamodb;
+pub extern crate aws_sdk_dynamodb;
```

Comment:
You might want to add the feature for this
```suggestion
#[cfg(feature = "dynamodb")]
pub extern crate aws_sdk_dynamodb;
```

Reply:
I thought it was for all lines below